qml/components-Sailfish/IconButton.qml

```diff
@@ -0,0 +1,30 @@
+import Sailfish.Silica 1.0
+import "." as BVApp
+
+IconButton {
+    function iconBy(type) {
```

Comment:
I'd suggest just to move the functions `iconBy(type)` into the according Themes and set the scaling factor from the outside, as it is now already.